pymc3/tuning/starting.py

```diff
@@ -96,10 +98,11 @@ def find_MAP(
     vars = inputvars(vars)
     disc_vars = list(typefilter(vars, discrete_types))
     allinmodel(vars, model)
-
+    start = copy.deepcopy(start)
     if start is None:
         start = model.test_point
     else:
+        start = {k: v for k, v in start.items()}
```

Comment:
We only need the deepcopy above

Reply:
Ohhh sorry, I just forgot to remove that🤦‍♂️